src/screens/VerificationScreen.js

```diff
@@ -1,50 +1,169 @@
 import * as React from 'react';
+import { useState, useRef } from 'react';
 
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import firebase from 'firebase/compat/app';
 import { shape, func, string } from 'prop-types';
 import {
+  Dimensions,
+  Pressable,
+  TouchableWithoutFeedback,
   Text,
   View,
   TextInput,
   Button,
+  Keyboard,
   StyleSheet,
   TouchableOpacity,
+  KeyboardAvoidingView
 } from 'react-native';
 import { Title } from 'react-native-paper';
 
 import ViewContainer from '../components/ViewContainer';
+import { Inter_200ExtraLight } from '@expo-google-fonts/inter';
 
 const styles = StyleSheet.create({
-  separator: {
-    marginVertical: 30,
-    height: 1,
-    width: '80%',
-    backgroundColor: '#eee',
+  heading: {
+    fontSize: 28,
+    fontWeight: '700',
+    marginTop: '6%',
   },
+  caption: {
+    marginTop: '2.3%',
+    marginRight: 12,
+    marginLeft: 12,
+    textAlign: 'center',
+    fontSize: 17,
+    color: '#777',
+  },
+  captionPhoneNumber: {
+    width: '100%',
+    display: 'flex',
+    textAlign: 'right',
+    marginRight: 12,
+    marginLeft: 12,
+    fontSize: 17,
+    color: 'black',
+  },
+  rowContainer: {
+    flexDirection: 'row'
+  },
+  input: {
+    textAlign: 'center',
+    marginBottom: 15,
+    marginRight: 7,
+    marginLeft: 7,
+    width: '12.85%',
+    height: 50,
+    fontSize: 18,
+    position: 'relative',
+    borderWidth: 2,
+    borderColor: '#52bd41',
+    borderRadius: 8,
+  },
+  inputUnfocused: {
+    textAlign: 'center',
+    marginBottom: 15,
+    marginRight: 7,
+    marginLeft: 7,
+    width: 40,
+    height: 50,
+    fontSize: 18,
+    position: 'relative',
+    borderWidth: 2,
+    borderColor: '#ADB0B9',
+    borderRadius: 8,
+  },
+  containerForVerficationInput: {
+    padding: '2.3%',
+    backgroundColor: "#ffffff",
+    flexDirection: 'row'
+  },
+  codeResent: {
```

Comment:
Try decreasing the space between the "CODE RESENT" text and "Didn't get a code?" text